lib/control-app.js

```diff
@@ -5,6 +5,10 @@ var express = require('express');
 var expstate = require('express-state');
 var browserify = require('browserify');
 var im = require('istanbul-middleware');
+var watchify = require('watchify');
+var _ = require('lodash');
```

Comment:
Can we use the specific lodash modules (https://www.npmjs.com/package/lodash.assign) versus just getting everything. Less likely to break something on an upgrade later.